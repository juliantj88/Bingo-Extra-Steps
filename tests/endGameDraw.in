1
test1
test2
1 R 5
2 U 1
5 R 5
4 U 5
0 B 2
0 U 2
3 B 1
0 Y 4
0 L 3
3 L 3
4 B 1
5 Y 2
0 U 4
1 Y 2
2 R 3
0 R 4
0 B B
0 Y B
4 U 5
2 R 4
0 Y 3
1 L 1
3 Y 3
0 R 2
5 Y 3
0 B B
0 U 2
0 R 3
2 Y 5
1 U 2
4 L 3
5 U 4
0 B 2
3 R 3
0 Y 5
0 R B
0 L 3
0 U 4
5 U 4
1 L 2
3 Y 5
0 U 4
4 L 2
2 R B
0 U B
0 L B
0 R 1
0 Y 3
0 B B
5