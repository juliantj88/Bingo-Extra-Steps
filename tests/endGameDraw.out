test1
test2
4
4
1
F...............
YYYY
LUUU
LUUU
RLLB
BBUU
ByruL
LBYRU
uLbyR
RUlby
yRulb
ByRUl
LByrU
uLbYR
rUlbY
YrUlb
.
..
YY.
....
.....
.
..
...
....
.....
       
       

RRLBBYRLLUYBURRBLRYRBL
0